### PR TITLE
pull out methods into functions

### PR DESCRIPTION
**Pull request by nickpell**
https://github.com/dimagi/commcare-hq/pull/24600#discussion_r295165155

### CODE REVIEW

**Review by czue — APPROVED** [review]
thanks for follow up!

File: corehq/apps/api/odata/serializers.py [hunk 1]
```diff
@@ -40,32 +39,33 @@ def to_json(self, data, options=None):
...
-        })
-
-    @memoized
-    def get_properties_to_include(self, domain, case_type):
-        case_type_to_properties = get_case_type_to_properties(domain)
-        return [
-            'case_name', 'case_type', 'date_opened', 'owner_id', 'backend_id'
-        ] + case_type_to_properties.get(case_type, [])
+
+def get_properties_to_include(domain, case_type):
```
Comment (czue) [inline corehq/apps/api/odata/serializers.py]: it looks like you removed the caching. assume that was intention.

**Comment by czue** [comment]
looked like an unrelated build failure so i kicked it

File: corehq/apps/api/odata/serializers.py (hunk 1 above)
Comment (nickpell) [reply corehq/apps/api/odata/serializers.py]: yes - I was using caching so the method would only get executed once per request, but now it's only called once without needing caching.

**Pull request merged**